### PR TITLE
Tests/build question

### PR DESCRIPTION
**Pull request by shaargtz**
# What does this PR do?
- Tests the buildQuestion function

**Related Issues**
- NA

---
**Merge Checklist**
- [x] Branch is up to date
- [x] No code style issues

### CODE REVIEW

File: src/test/java/com/google/sps/UtilityTest.java [hunk 1]
```diff
@@ -115,4 +122,50 @@ public void nullValue() {
...
     Assert.assertEquals(actual, expected);
   }
+
+  /** 
+   * Tests for buildQuestion function.
+   * Testing for null input is out of scope since it never happens.
+   */
+  @Test
+  public void normalResult() {
+    ResultSet testResultTest = mock(ResultSet.class);
```
Comment (OUMontiel) [inline src/test/java/com/google/sps/UtilityTest.java:132]: Why is test twice in the name? Maybe you meant `testResultSet`?

File: src/test/java/com/google/sps/UtilityTest.java [hunk 2]
```diff
@@ -115,4 +122,50 @@ public void nullValue() {
 
     Assert.assertEquals(actual, expected);
   }
+
+  /** 
+   * Tests for buildQuestion function.
+   * Testing for null input is out of scope since it never happens.
+   */
+  @Test
+  public void normalResult() {
```
Comment (gloriapark) [inline src/test/java/com/google/sps/UtilityTest.java:131]: Same comment here as previous test PR: 

A common style practice for naming java unit test functions it to include the method being tested, the test condition, and the expected outcome, so something like: buildQuestion_normalResult_success(). This becomes especially useful for readability when you have a lot of different functions being tested within the same test file (ie. this UtilityTest file).

Doing this makes it really explicitly clear what is being tested, how it's being tested, and what we expect to see. Consider updating your methods to also follow this pattern!

It's not technically part of the public github Java style guide, so I'll leave it up to you - but a good habit to have :)

File: src/test/java/com/google/sps/UtilityTest.java (hunk 1 above)
Comment (shaargtz) [reply src/test/java/com/google/sps/UtilityTest.java:132]: oh yeah ahahahah little typo

File: src/test/java/com/google/sps/UtilityTest.java (hunk 2 above)
Comment (shaargtz) [reply src/test/java/com/google/sps/UtilityTest.java]: thanks!

**Pull request merged**